test/lib/ufe/testChildFilter.py

```diff
@@ -92,7 +92,7 @@ def testFilteredChildren(self):
         ball3Prim = mayaUsd.ufe.ufePathToPrim(ball3PathStr)
         ball3Prim.SetActive(False)
 
-        # Props should now have 5 children and ball3 should not be one of them.
+        # Props children should have 5 children and ball3 should not be one of them.
```

Comment:
Why did you remove "now". The comment was correct. At line 84 we test that there are 6 children and now we are testing that there are 5.

Reply:
I took multiple passes to update the test and did not realized I dropped a word there.